.travis.yml

```diff
@@ -1,12 +1,23 @@
 language: go
 services:
-    - redis-server
+- redis-server
 go:
-  - 1.4
-install: ./scripts/build-setup.sh
-script: 
-    - ./scripts/build-build.sh
-    - ./scripts/test.sh
+- 1.4
+install: sleep 0 # Travis does go get -t -v- ./... which we don't want.
+script:
+- ./scripts/build.sh
```

Comment:
:+1: do you recall in particular why not? Were there specific dynamically linked shared libs that were missing?